website/courses/druid-monitoring/druid-monitoring-conclusion.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 4
+title: Conclusion
+---
+
+## Conclusion
+
+<!--Need video-->
+
+## Join the community
```

Comment:
@techdocsmith I think we should just take out the join the community section from each course and add it as a standalone top level page in the left nav